jha/src/views/enrolment/ChildInfoPage.vue

```diff
@@ -43,35 +50,39 @@
               :id="'child-first-name-' + index"
               className='mt-3'
               maxlength="18"
+              @blur="handleBlurField(v.firstName)"
               v-model='child.firstName' />
             <div class="text-danger"
               v-if="v.firstName.$dirty && !v.firstName.required"
               aria-live="assertive">First name is required.</div>
             <div class="text-danger"
               v-if="v.firstName.$dirty && v.firstName.required && !v.firstName.nameValidator"
-              aria-live="assertive">First name must begin with a letter and cannot include special characters except hyphens, periods, apostrophes and blank characters.</div>
+              aria-live="assertive">First name must begin with a letter and cannot include special characters except hyphens, periods, apostrophes and blank characters.<br/>First name must be a letter.</div>
```

Comment:
I'm confused at this. If the first name is longer than one letter, then it wouldn't be "a letter" like this added text says it is. Can you tell me more about the purpose this text serves?

Reply:
It's total nonsense but they don't want any change to the error messaging from the existing enrollment form, which has this text. You'll see the same in the spouse and personal info pages of JHA. It's wrong and it bugs me too but this is how they want it (for now)

Reply:
I see! Well, I can't fault you for following the requirements. Other than that, looks good to me